### PR TITLE
Drop old Mypy ignores

### PR DESCRIPTION
**Pull request by Dreamsorcerer**

### CODE REVIEW

**Comment by Dreamsorcerer** [comment]
Remaining 2 errors will hopefully be solved by https://github.com/gabrieldemarmiesse/python-on-whales/pull/580

File: requirements/lint.in [hunk 1]
```diff
@@ -1,6 +1,8 @@
+aiodns
```
Comment (webknjaz) [inline requirements/lint.in:1]: Looks like the pins need updates too?

File: tests/test_resolver.py [hunk 2]
```diff
@@ -289,13 +289,6 @@ async def test_default_loop_for_threaded_resolver(loop: Any) -> None:
     assert resolver._loop is loop
 
 
-@pytest.mark.skipif(aiodns is None, reason="aiodns required")
-async def test_default_loop_for_async_resolver(loop: Any) -> None:
```
Comment (webknjaz) [inline tests/test_resolver.py:293]: Why does the test need to be removed?

Comment (Dreamsorcerer) [reply tests/test_resolver.py:293]: It's only testing the _loop attribute, which I just removed, as it has no real purpose.

**Pull request merged**